components/mailer.rst

```diff
@@ -16,8 +16,128 @@ Installation
 
 .. include:: /components/require_autoload.rst.inc
 
+
+Introduction
+------------
+
+
 Usage
 -----
 
-We're currently working on the documentation of this component that was just
-added to Symfony. We'll publish it in a few days.
+The Mailer component has two main classes: a ``Transport`` and the ``Mailer`` itself::
+
+    use Symfony\Component\Mailer\Mailer;
+    use Symfony\Component\Mailer\Transport\Smtp\SmtpTransport;
+
+    $transport = new SmtpTransport('localhost');
+    $mailer = new Mailer($transport);
+    $mailer->send($email);
+
+Refer :doc:`Mime component </components/mime>` how to create `$email` object.
```

Comment:
```suggestion
The `$email` object is created via the :doc:`Mime component </components/mime>`.
```

Reply:
If everyone needs that, I will make the change.

Reply:
Let's merge this suggestion